homeassistant/components/fitbit/sensor.py

```diff
@@ -714,7 +735,7 @@ async def async_update(self) -> None:
 
 
 class FitbitBatterySensor(CoordinatorEntity, SensorEntity):
```

Comment:
```suggestion
class FitbitBatterySensor(CoordinatorEntity[FitbitDeviceCoordinator], SensorEntity):
```

Reply:
Sorry not sure what happened here -- i think i accidentally lost my commits that also did this when merging the tpo suggestion. Thanks for just fixing.